app/views/candidate_interface/degrees/degree/new_country.html.erb

```diff
@@ -9,8 +9,15 @@
     <%= f.govuk_radio_buttons_fieldset :uk_or_non_uk, legend: { text: 'Which country was the degree from?', size: 'l' } do %>
       <%= f.govuk_radio_button :uk_or_non_uk, 'uk', label: { text: 'United Kingdom' }, link_errors: true %>
       <%= f.govuk_radio_button :uk_or_non_uk, 'non_uk', label: { text: 'Another country' } do %>
-        <%= f.govuk_collection_select :country, select_degree_country_options, :id,
```

Comment:
`select_degree_country_options` can be removed from the helper

Reply:
Cool, happy to removed then :)